packages/php-wasm/node/src/test/php.spec.ts

```diff
@@ -1262,6 +1262,45 @@ describe.each(SupportedPHPVersions)('PHP %s', (phpVersion) => {
 			expect(json).toHaveProperty('CONTENT_TYPE', 'text/plain');
 			expect(json).toHaveProperty('CONTENT_LENGTH', '15');
 		});
+
+		it('Should have appropriate SCRIPT_NAME, SCRIPT_FILENAME and PHP_SELF entries in $_SERVER when serving request', async () => {
+			php.writeFile(
+				'/php/index.php',
+				`<?php echo json_encode($_SERVER);`
+			);
+
+			const response = await php.request({
+				url: '/',
+				method: 'GET',
+			});
+
+			const json = response.json;
+
+			expect(json).toHaveProperty('REQUEST_URI', '/');
+			expect(json).toHaveProperty('SCRIPT_NAME', '/index.php');
```

Comment:
Let's also add a test to see what happens when the file is called `test.php` is in a subdirectory, e.g. `/php/subdirectory/test.php`

Reply:
I suppose I should add this test : 

```
php.mkdir('/php/subdirectory');

php.writeFile(
	`/php/subdirectory/test.php`,
	`<?php echo json_encode($_SERVER);`
);

const response = await php.run({
	scriptPath: '/php/subdirectory/test.php',
	relativeUri: '/test',
	method: 'GET',
});
```

But it fails, using `run` will weirdly always lead to `SCRIPT_NAME` and `SCRIPT_FILENAME` be equal to `REQUEST_URI` [ `/test` here ] like `wasm_server_context->document_root` or `wasm_server_context->path_translated` is always NULL in `php_wasm.c`. 

I have to investigate.

Reply:
@adamziel I added it. Is this what you expected ?

Reply:
@adamziel I added the test you asked. Is it what you expected ?